### PR TITLE
add useAtomValue

### PR DESCRIPTION
**Pull request by Aslemammad**
I'll do useAtomValue( suggested by sophia) tests and ... in other PR. #212

### CODE REVIEW

**Review by dai-shi — COMMENTED** [review]
Thanks for your work. Would you update docs/utils.md?
I think it should be colocated with useUpdateAtom

**Comment by Aslemammad** [comment]
@dai-shi Can I do that in different PR, I think it would be cleaner.

**Comment by merisbahti** [comment]
In this PR: https://github.com/pmndrs/jotai/pull/193

I both added util, updated docs, and updated the `.size-snapshot.json` in one. I think there's merit to make these kind of changes in a consistent package, where all related info is updated in one PR (but ofc, can be multiple commits within that PR).

Just giving my 2 eurocent :)

**Comment by Aslemammad** [comment]
@merisbahti Thanks for your suggestion. done.

File: docs/utils.md [hunk 1]
```diff
@@ -23,6 +23,30 @@ const Controls = () => {
...
+Ref: https://github.com/pmndrs/jotai/issues/212
+
+```js
+import { atom, Provider, useAtom } from "jotai"
+import { useAtomValue } from "jotai/utils"
+
+const countAtom = atom(0)
+
+const Counter = () => {
+  const [, setCount] = useAtom(countAtom)
```
Comment (merisbahti) [inline docs/utils.md:37]: Maybe this one isn't needed, and we can comment out the one below?

File: src/utils.ts [hunk 2]
```diff
@@ -5,4 +5,5 @@ export { atomWithReducer } from './utils/atomWithReducer'
 export { atomFamily } from './utils/atomFamily'
 export { useSelector } from './utils/useSelector'
 export { useAtomCallback } from './utils/useAtomCallback'
+export { useAtomValue } from './utils/useAtomValue'
```
Comment (dai-shi) [inline src/utils.ts:8]: This is just a minor thing, and we never had a strict rule so far.
But, how about moving this in L2, to be consistent with the order in docs?

Comment (Aslemammad) [reply src/utils.ts]: @dai-shi done.

**Review by dai-shi — APPROVED** [review]

**Comment by Aslemammad** [comment]
@dai-shi Thanks.

**Pull request merged**